tests/test_helper.py

```diff
@@ -0,0 +1,23 @@
+from tests import JinaTestCase
```

Comment:
I think these tests are a great idea, we should also add a `test` for `batching` decorator